concent_api/core/message_handlers.py

```diff
@@ -1483,3 +1487,25 @@ def validate_that_golem_messages_are_signed_with_key(
                 f'public key {public_key}.',
                 error_code=ErrorCode.MESSAGE_SIGNATURE_WRONG,
             )
+
+
+def validate_if_given_object_is_given_enum_instance(
```

Comment:
1. Move this function to validation.py please.
2. Name of the function sound weird. Can't it be just `validate_if_given_object_is_enum_instance` or `validate_if_given_object_is_correct_enum_instance` or `validate_if_given_object_is_expected_enum_instance`?